README.md

```diff
@@ -28,11 +28,11 @@ There has been increasing [developer](https://github.com/GoogleChromeLabs/privac
 
 ### [Example 1](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/issues/124)
 
-A developer embeds chat.com on two of their sites site-a.com and site-b.com. chat.com uses Shared Workers to maintain a user session.
+A developer embeds chat.com on two of their sites site-a.com and site-b.com. chat.com uses IndexedDB to maintain a user session.
```

Comment:
As discussed offline, based on the fact that the primary use cases here are for Shared Workers we should consider following up with a solution for that as a V2. So, I do think it makes sense to keep SharedWorkers in the examples here, as replacing it with IndexedDB feels a bit misrepresentative to me.